ParallelWithTestNG/pom.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why not put in `java` folder ?

Reply:
Should update `java/README.md` for how to execute this test: how to build, how to run, how to provide api key, etc.